provd/internal/services/manager.go

```diff
@@ -0,0 +1,40 @@
+// Package services mediates all the business logic of the application via a manager.
+package services
+
+import (
+	"context"
+
+	"github.com/canonical/ubuntu-desktop-provision/provd"
+	"github.com/canonical/ubuntu-desktop-provision/provd/internal/services/hello"
+	"github.com/ubuntu/decorate"
+	"google.golang.org/grpc"
+	"log/slog"
+)
+
+// Manager mediate the whole business logic of the application.
```

Comment:
```suggestion
// Manager mediates the whole business logic of the application.
```

(should be a conjugated verb)